vendor/src/github.com/docker/engine-api/types/registry/registry.go

```diff
@@ -82,7 +82,7 @@ type SearchResult struct {
 	IsOfficial bool `json:"is_official"`
 	// Name is the name of the repository
 	Name string `json:"name"`
-	// IsOfficial indicates whether the result is trusted
+	// IsTrusted indicates whether the result is trusted
```

Comment:
This is the comment for the `IsTrusted` flag, not `IsOfficial`. The comment referenced the incorrect variable.